cmd/create/cluster/cmd.go

```diff
@@ -3264,21 +3265,25 @@ func handleOidcConfigOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool, is
 	return oidcConfig
 }
 
-func filterPrivateSubnets(initialSubnets []*ec2.Subnet, r *rosa.Runtime) []*ec2.Subnet {
+func filterPrivateSubnets(initialSubnets []ec2types.Subnet, r *rosa.Runtime) []ec2types.Subnet {
+	initialSubnetPointers := make([]*ec2types.Subnet, len(initialSubnets))
```

Comment:
If we change the signature of `FetchPublicSubnetMap` `FetchPublicSubnetMap(subnets []ec2types.Subnet)`  I think we won't need the casting here.
Inside the function we are using only the subnet IDs, therefore, we can also send a slice of subnets, WDYT?

Reply:
@den-rgb can we change `FetchPublicSubnetMap` signature and remove the casting?

Reply:
no we dont i have removed it now